### PR TITLE
regular container update 2023-11 for other images from platform team

### PR DESCRIPTION
**Pull request by umezawatakeshi**
part of https://github.com/cybozu-private/neco-tasks/issues/39

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

File: pomerium/BRANCH
Comment (yamatcha) [inline pomerium/BRANCH:1]: Why BRANCH is deleted?

Comment (umezawatakeshi) [reply pomerium/BRANCH:1]: It's not used

Comment (umezawatakeshi) [reply pomerium/BRANCH:1]: Sorry, it means that we do not specify container images in the form of branch-tag i.e. `quay.io/cybozu/pomerium:0.22`

Comment (yamatcha) [reply pomerium/BRANCH:1]: @umezawatakeshi 
OK. I understand. 
Please update the maintenance procedure.
https://github.com/cybozu/neco-containers/blob/685ebf867d038f85b3d2b146facb173ea2258a3d/maintenance.md?plain=1#L720

**Review by yamatcha — DISMISSED** [review]

**Review by yamatcha — APPROVED** [review]

**Pull request merged**